cegs_portal/search/templates/search/v1/search_results.html

```diff
@@ -127,6 +127,31 @@
         0% {transform: scale(1);}
         100% {transform: scale(0.9);}
     }
+
+    .search-results-data-table td {
+        padding: 4px;
+        border-bottom: 1px solid lightgrey;
+        font-size: small;
+    }
+
+    .search-results-table-title  {
+        justify-content: left;
+        font-family: sans-serif;
+        font-size: 20px;
+        font-weight: bold;
+        color: rgb(107, 107, 107);
+    }
+
+    #table-column-hidden {
+        display: none;
+    }
+
+    #spinner {
```

Comment:
I think an id is fine for this, but you should use a more specific name like `dna-feature-loading-spinner`